spec/ibandit/swedish_details_converter_spec.rb

```diff
@@ -21,6 +28,16 @@
         its([:swift_account_number]) { is_expected.to eq('00000012810105723') }
       end
 
+      context 'with a separate branch code' do
+        let(:branch_code) { '1281' }
+        let(:account_number) { '0105723' }
+
+        its([:account_number]) { is_expected.to eq('105723') }
```

Comment:
I'm unsure about this case - we lose the leading zero from the serial number here, whereas we kept it when we passed in a combined clearing code/serial number (the `swift_account_number` is different too).  

The entry in `swedish_bank_lookup.yml` has `zerofill_serial_number` set to `false`.  Any idea what we should be doing here?